editor/src/clj/editor/code/view.clj

```diff
@@ -3810,6 +3811,7 @@
     (let [find-case-sensitive-setter (make-property-change-setter view-node :find-case-sensitive?)
           find-whole-word-setter (make-property-change-setter view-node :find-whole-word?)
           font-size-setter (make-property-change-setter view-node :font-size)
+          zoom-on-mouse-wheel-setter (make-property-change-setter view-node :zoom-on-mouse-wheel?)
```

Comment:
Do we need this?

Reply:
I initially thought we did and forgot to remove it. If we the pref within the wheel handler, we won't have to reopen the view to apply the change.